tests/kafkatest/__init__.py

```diff
@@ -23,4 +23,4 @@
 # Instead, in trunk, the version should have a suffix of the form ".devN"
 #
 # For example, when Kafka is at version 0.9.0.0-SNAPSHOT, this should be something like "0.9.0.0.dev0"
-__version__ = '0.10.0.0.dev0'
+__version__ = '0.10.0.1.dev0'
```

Comment:
Yeah, I think you're right -- the `dev0`, including the number, are so we can release intermediate snapshots to PyPI if needed/helpful such that they stop being a moving target. (In practice, we haven't actually released to PyPI yet; however, we may want to figure out how to include that in the release process moving forward).

However, I'll let @granders have final say here since he was the one who figured out how the python versioning should work.